packages/flutter_tools/lib/src/features.dart

```diff
@@ -296,6 +305,30 @@ const Feature flutterFuchsiaFeature = Feature(
   ),
 );
 
+const Feature flutterCustomDevicesFeature = Feature(
+  name: 'alpha-quality support for custom, individually configured devices',
```

Comment:
I would leave out the branding designation. "Early support for custom device types"